CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Change Log
 
 ## [Unreleased]
-### Added
+### Added
```

Comment:
Don't know why this line has change, but I guess the spaces changes can break the markdown. We'll see.